patches/net/minecraft/world/item/crafting/RecipeManager.java.patch

```diff
@@ -9,6 +9,14 @@
          );
          List<RecipeHolder<?>> list = new ArrayList<>(sortedmap.size());
          sortedmap.forEach((p_379232_, p_379233_) -> {
+@@ -93,6 +_,7 @@
+             .stream()
+             .map(p_380840_ -> new RecipeManager.IngredientCollector(p_380840_.getKey(), p_380840_.getValue()))
+             .toList();
++        this.recipes.order(); // Neo: order recipes by priority.
```

Comment:
Can you move this to the top of the method? It's a bit of a nitpick but I really don't see why you chose to place it there.

Reply:
The definition of `list` and `list1` is already part of the iteration over recipes for me since their first usage is in the `forEach`.